src/routes/chart/get.ts

```diff
@@ -33,6 +34,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
       pinHashId: Joi.string().required().example('e13d57'),
       quantityHashId: Joi.string().required().example('sajia1'),
     })).required(),
+    highResolution: Joi.boolean().required(),
```

Comment:
Can you replace `.required()` with `.default(false)`? That we we ensure backwards compatibility (unrefreshed browsers and people using our sdk directly will not send highResolution).

Reply:
Updated